ios/RNAppAuth.m

```diff
@@ -407,4 +410,51 @@ - (NSDictionary*)formatRegistrationResponse: (OIDRegistrationResponse*) response
              };
 }
 
+- (NSString*)getErrorCode: (NSError*) error defaultCode: (NSString *) defaultCode {
```

Comment:
Not an Objective-C programmer, so just copied the example from Apple documentation https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ErrorHandlingCocoa/CreateCustomizeNSError/CreateCustomizeNSError.html#//apple_ref/doc/uid/TP40001806-CH204-BAJCIGIA. Sorry if this is against some conventions used by the library.